### DISCUSSION TITLE
Theme colors not showing up

### DISCUSSION THREAD

**Discussion by brysbroughton** [vaga/hugo-theme-m10c#82] [Q&A] [answered]
HI, I like your theme. However, I am getting some issues changing the theme colors. Am I missing something?

![image](https://user-images.githubusercontent.com/5316322/159555587-b4970f2c-d5d2-42a2-927d-260eb3341c18.png)

Cheers

**Comment by vaga** [comment, marked as answer]
Hi,
You should use Hugo extended version.
https://github.com/vaga/hugo-theme-m10c#styling

:)
